builder/dockerfile/copy.go

```diff
@@ -200,27 +209,64 @@ func (o *copier) calcCopyInfo(origPath string, allowWildcards bool) ([]copyInfo,
 	return newCopyInfos(newCopyInfoFromSource(o.source, origPath, hash)), nil
 }
 
+func validateCopySourcePath(imageSource *imageMount, origPath, platform string) error {
```

Comment:
Did this need to be moved out of the windows only file? I understand some linux functionality needs to be moved so it can be used with Windows builds, but the other direction shouldn't be needed (unless we are doing WCOL 😱)

Reply:
I see. I can revert it back to a Windows implemenation for LCOW/WCOW and a no-op Unix implementation.